clio-transfer-model/src/main/scala/org/broadinstitute/clio/transfer/model/SemiAutoClioIndex.scala

```diff
@@ -60,16 +60,25 @@ case object GvcfIndex
   override val elasticsearchIndexName = "gvcf-v2"
 }
 
-case object WgsUbamIndex
+// Temporary trait to remain until WgsUbam fully phased out
+trait BackCompatibleUbamIndex
     extends SemiAutoClioIndex[
       UbamKey,
       UbamMetadata,
       UbamQueryInput
     ] {
+  override val name: String = "Ubam"
+  override val elasticsearchIndexName: String = "ubam"
```

Comment:
I don't think we can change this value yet. It's used to derive the name of the directory in GCS storing all of the existing ubam upserts, so if we change it the recovery code won't be able to find them.